keyboards/ymd75/keymaps/via/config.h

```diff
@@ -0,0 +1 @@
+#define DYNAMIC_KEYMAP_LAYER_COUNT 3
```

Comment:
Still in the wrong place, it should be in ymd75/config.h.

Reply:
It's done)